.github/workflows/tests.yaml

```diff
@@ -829,7 +829,29 @@ jobs:
           password: ${{secrets.DOCKER_PASSWORD}}
       - name: Run chaos test
         run: ./multi_tenancy_concurrent_importing.sh
+  multi_tenancy_activate_deactivate_v1_24:
+    if: ${{ github.event.inputs.weaviate_max_major_version_test_set == 1.24 }}
```

Comment:
**praise (non-blocking):**  i love this, because it will not double run the step from 1.24+